### PR TITLE
Add ConstructRemoveAuthorizedParty

### PR DESCRIPTION
**Pull request by jleni**
Add test

### CODE REVIEW

File: construction_impl.go [hunk 1]
```diff
@@ -371,6 +371,118 @@ func (r RosettaConstructionFilecoin) ConstructSwapAuthorizedParty(request *SwapA
...
+		bufRemoveSigner := new(bytes.Buffer)
+		err = removeSignerParams.MarshalCBOR(bufRemoveSigner)
+		if err != nil {
+			return "", err
+		}
+
+		params := &multisigV1.ProposeParams{
+			To:     to,
+			Value:  types.NewInt(0),
+			Method: builtinV1.MethodsMultisig.SwapSigner,
```
Comment (arajasek) [inline construction_impl.go:424]: Should be calling `RemoveSigner`

File: construction_impl.go [hunk 2]
```diff
@@ -371,6 +371,118 @@ func (r RosettaConstructionFilecoin) ConstructSwapAuthorizedParty(request *SwapA
...
+		bufRemoveSigner := new(bytes.Buffer)
+		err = swapSignerParams.MarshalCBOR(bufRemoveSigner)
+		if err != nil {
+			return "", err
+		}
+
+		params := &multisigV2.ProposeParams{
+			To:     to,
+			Value:  types.NewInt(0),
+			Method: builtinV2.MethodsMultisig.SwapSigner,
```
Comment (arajasek) [inline construction_impl.go:451]: Should be calling `RemoveSigner`

File: construction_api.go [hunk 3]
```diff
@@ -129,3 +135,17 @@ type SwapAuthorizedPartyRequest struct {
...
 	Params   SwapAuthorizedPartyParams `json:"params"`
 }
+
+// RemoveAuthorizedPartyParams defines the params
+type RemoveAuthorizedPartyParams struct {
+	ToRemove string `json:"toRemove"`
+	Decrease bool   `json:"decrease"`
+}
+
+// RemoveAuthorizedPartyRequest defines the input to ConstructSwapAuthorizedParty
```
Comment (arajasek) [inline construction_api.go:145]: Comment is incorrect, should be `ConstructRemoveAuthorizedParty`

**Review by arajasek — COMMENTED** [review]
looks like some copy-pasting bugs

File: construction_impl.go (hunk 1 above)
Comment (qiwu7) [reply construction_impl.go:424]: good 👁️ s. @raynaudoe can we fix it?

Comment (jleni) [reply construction_impl.go:424]: oopsss  :+1: 
we definitely need a bit more coverage here

**Pull request merged**